### ISSUE TITLE
Accessibility: "go to latest message" in conversation not labeled when using Voice Over

### ISSUE THREAD

**Issue by Falcopus1986** [signalapp/Signal-iOS#4803]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
When Voice over is turned on and you go back in a conversation the button that appears to go to the latest message in a conversation is unlabeled.
### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug
- Turn Voice-over on
- Open a conversation
- Go back in to earlier messages in the conversation
- Until the "go to last message" button appears 
**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
- "go to latest message" button is unlabeled also in other languages like Dutch this button is unlabeled

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
- Name of unlabe### Screenshots

### Device info

**Device**: iPhone XS

**iOS version**: 14.3

**Signal version:** 5.2.1.4
